### DISCUSSION TITLE
Invalid request for route publish_po_message: OpenAPI spec contains no such operation

### DISCUSSION THREAD

**Discussion by shubhangicatch** [OAI/OpenAPI-Specification#3014] [General]
Hi ,

Anyone have idea why i am getting this error .

Thanks
Shubhangi

**Comment by flaksp** [comment]
A specification document called OpenAPI is developed in this repository. It's a paper that describes how to document your API in the standardised format most tools will understand. You may browse it at https://spec.openapis.org/oas/latest.html

The question you are asking relates to some tool you are using. OAI organisation does not belong to any tools. You should consider to ask this question in a repository of the tool you are using.

**Comment by handrews** [comment]
As noted above, this should be directed to the tooling vendor.  Closing.
